src/script/conversation/EventMapper.ts

```diff
@@ -421,6 +427,10 @@ export class EventMapper {
     return messageEntity;
   }
 
+  _mapEventCallingTimeout({data, time}: EventRecord) {
+    return new CallingTimeoutMessage(data.reason, +time);
```

Comment:
```suggestion
    return new CallingTimeoutMessage(data.reason, +time);
```

Whats the trick with the `+` here?

Reply:
converts time from string to number

Reply:
Please use `parseInt` for this.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10709/commits/d5499896f8af1e73011a0af371dd0f4c1f6ecd0e